sql/2022/jamstack/distribution_age_headers.sql

```diff
@@ -0,0 +1,33 @@
+-- getting ages to calculate median age (or any other percentage threshold)
+-- we're using 2022-07-01 run as the baseline because values change every year

```

Comment:
Shouldn't this only be looking at the HTML page request, rather than all requests?